web/source/build.bat

```diff
@@ -1,198 +0,0 @@
-@echo off
```

Comment:
Now that build.bat is removed, you can strike the comment in build.sh line 5 about having to keep changes in sync with build.bat